srcpkgs/datovka/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'datovka'
+pkgname=datovka
+version=4.21.1
+revision=1
+archs="~*-musl"
+build_style=qmake
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="libdatovka-devel qt5-websockets-devel qt5-svg-devel libxml2-devel"
```

Comment:
```suggestion
makedepends="libdatovka-devel qt5-websockets-devel qt5-svg-devel libxml2-devel
 quazip-devel"
```

Just noticed that it uses bundled quazip (see build logs) when the repository already has one.

Could you try to build with `quazip-devel`?

Reply:
It builds locally without issues.